oauth2server/apps/accounts/views.py

```diff
@@ -81,10 +80,28 @@ def put(self, request):
             # need to tell the other system we are not capable of creating the resource.
             raise DatabaseFailureException
 
+        # if account_locked attribute set to False, we want to unlock the account
+        try:
+            # Check to see if the POST contains the account_locked flag.
+            # If so we unlock account and reset failed log-ins.
+            if request.POST.get('account_locked') == 'False':
```

Comment:
Can we not merge both try excepts into one? Probably only need the try except around the `request.user.save()`